packages/gatsby-telemetry/src/error-helpers.js

```diff
@@ -1,11 +1,18 @@
+const { sep } = require(`path`)
+
 // Removes all user paths
-const cleanPaths = str => {
-  // TODO: Recursively remove path sections
-  const localPathRegex = new RegExp(
-    process.cwd().replace(/[-[/{}()*+?.\\^$|]/g, `\\$&`),
```

Comment:
should this be also a helper function because we use the same function in a test at:

https://github.com/gatsbyjs/gatsby/blob/5e0217e63609f6081acce4f80749474373b6ddf2/packages/gatsby-telemetry/src/__tests__/error-helpers.js#L45